localization/languages/ko.json

```diff
@@ -97,53 +97,53 @@
     "timeRangeWeek": "지난 주",
     "timeRangeMonth": "지난 달",
     "timeRangeYear": "작년",
-    "timeRangeLongerAgo": "오래전",
+    "timeRangeLongerAgo": "호랑이 담배 피던 시절",
```

Comment:
If I understand correctly, this is an idiomatic expression - would a more literal statement be better here?

Reply:
Korea enjoys including funny phrases in its programs.
The phrase has exactly the same meaning as it did a longer ago. this will be an interesting point.
I prefer not to edit it, but if you want to edit it, I'll accept it.

Reply:
Let's go with the boring version, it's important to have consistency between the different languages the program can run in. Also, I asked a friend who speaks Korean for their opinion, and they agreed the boring version was better.